specs/core/0_beacon-chain.md

```diff
@@ -250,15 +250,15 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_VOLUNTARY_EXITS` | `2**4` (= 16) |
 | `MAX_TRANSFERS` | `0` |
 
-### Signature domain types
+### Domain types
```

Comment:
I'm a little bit unsure about conflating the type of signature and seed domain. I understand the intuition, but the usage of `DOMAIN_BEACON_ATTESTER` for both an attestation signature and committee shuffling seed domain is a bit dirty.

Thoughts on having both "Signature" and "Seed Domains"?

Reply:
The only reason I am for the singular domain type is to be minimally invasive to the phase 0 spec..

Reply:
I'd say merging the two is fine. Semantically, the "lifecycle" of a validator role (beacon/shard attester/proposer) is to first get sampled for that role and then sign a unique message under that specific role. So semantically it makes sense to merge the seed and signature domains.

Avoiding two types (`SeedDomainType` and `SignatureDomainType`), four extra constants, as well as verbosity (e.g. `SIGNATURE_DOMAIN_BEACON_ATTESTER` as opposed to just `DOMAIN_BEACON_ATTESTER`) is also nice.